src/s3/S3Agent.ts

```diff
@@ -1,160 +1,16 @@
-import { fromEnv } from '@aws-sdk/credential-providers';
-import {
-  CopyObjectCommand,
-  DeleteObjectCommand,
-  GetObjectCommand,
-  HeadObjectCommand,
-  ListObjectsV2Command,
-  PutObjectCommand,
-  S3Client,
-} from '@aws-sdk/client-s3';
 import { Buffer } from 'buffer';
+import { S3Controller } from './S3Controller';
 
-enum Endpoints {
-  LOCAL = 'http://localhost:9000/proavalon/',
-  STAGING = 'https://s3-staging.proavalon.com/file/proavalon-staging/',
+enum FolderName {
+  APPROVE = 'approved_avatars',
+  PENDING = 'pending_avatars',
 }
```

Comment:
```
interface S3AvatarKeys {
  resLink: string;
  spyLink: string;
}
```

Let's create a this guy, and then make all 3 methods related to avatar requests return and take in this interface. 

![Uploading image.png…]()